[16.0][MIG] sale_manual_delivery

nguyenhk99 opened the pull request.

nbessi committed:
[FIX] sale cancel reason dependence
(../trunk-generic/ rev 32.1.1)

Commit:
[FIX] sale_cancel_reason: remove dependency on sale_stock
(../trunk-generic/ rev 32.1.2)

Commit:
[MRG] from master branch
(../trunk-generic/ rev 29.1.1)

gurneyalex committed:
spellcheck

eLBati committed:
[FIX] repository PEP8

eLBati committed:
[FIX] Still some PEP8 erros

pedrobaeza committed:
- Set as uninstallable and moved to __unported__ dir all modules.
- Update travis.yml to 8.0

yvaucher committed:
reactivate sale_cancel_reason

yvaucher committed:
port models and method to api 8.0

yvaucher committed:
Improve override of cancel button to ease inheritance of view
Only hide button instead of renaming it

rdeheele committed:
[FIX] typo on class name

gurneyalex committed:
Add OCA as author of OCA addons

In order to get visibility on https://www.odoo.com/apps the OCA board has
decided to add the OCA as author of all the addons maintained as part of the
association.

mlaitinen committed:
Added Finnish translations

dhp-denero committed:
[FIX] cancel button of orignal view should be hide after installation of sale_cancel_reason module

yvaucher committed:
Add missing default oca icons

sysadminmatmoz committed:
Slovene translations added

yvaucher committed:
bump version of sale_cancel_reason

oca-transbot committed:
OCA Transbot updated translations from Transifex

pedrobaeza committed:
[MIG] Make modules uninstallable

sbidoul committed:
[FIX] remove en.po that was erroneously created by transbot

pedrobaeza committed:
[MIG] Rename manifest files

leemannd committed:
base module & sales_team

Add base for wizard

wizard for manual procurements

Update base computation

update to OCA guidelines

jcoux committed:
Add action_manual_procurement_wizard

Add action_manual_procurement_wizard in view

leemannd committed:
flake8 & OCA standards

jgrandguillaume committed:
[IMP] report manual procurement on SO from sale team

[ADD] Translation and .pot file

jgrandguillaume committed:
[TYP] Change manual proc for manual delivery

jgrandguillaume committed:
[IMP] Toggle manual,standard as a button in SO + UI

[IMP] Update translation files

[IMP] lot's of functionnal improvments

Don't add already planned or shipped lines in wizard
Ensure you have one picking per manual delivery
Forbid to add lines or ship more than ordered
Planned date is required
Improve views, naming and buttons
Better Readme
Default Carrier set from SO

[FIX] Do not add product of type 'service' in manual delivery creation

jgrandguillaume committed:
[ADD] Tests on various use cases

mdietrichc2c committed:
Regroup picking per date_planned / carrier + correct wizard action from SO

Add product name to manual delivery lines

vrenaville committed:
[FIX] picking split criteria

[FIX] call super in case of standard delivery

[FIX] when multiple picking candidate available take the right one with date corresponding

[FIX] revert of renaming field

Add SO line description

kittiu committed:
[MIG] sale_cancel_reason Migration to 10.0 (#477)

* [MIG] sale_cancel_reason Migration to 10.0

* Follow coding standard

* Convert test to unittest2

ysantiago committed:
product_supplierinfo_for_customer_sale: New module.

[REF] product_supplierinfo_for_customer_sale: Changes in test.

[REF] product_supplierinfo_for_customer_sale: Complete README.rst

ysantiago committed:
product_supplierinfo_for_customer_sale: Return all result in name search, products found by default code and customer code

[REF] product_supplierinfo_for_customer_sale: Add test to find more than one product in name_search

[REF] product_supplierinfo_for_customer_sale: In test find one product from customer code and other one from product default code.

[REF] product_supplierinfo_for_customer_sale: Changes in name_search to work exactly with number of limit defined.

[REF] product_supplierinfo_for_customer_sale: Changes in name_search

Commit:
add Delivery Address to wizard. by default selec SO partner address.

AaronHForgeFlow committed:
[UPT]rename module product_customer_code

sbidoul committed:
remove obsolete .pot files [ci skip]

oca-travis committed:
[UPD] Update sale_cancel_reason.pot

MiquelRForgeFlow committed:
[FIX] Pass customer code and min qty to sale order line

MiquelRForgeFlow committed:
[ADD] Tests

oca-travis committed:
[UPD] Update product_supplierinfo_for_customer_sale.pot

xavierjimenez committed:
[MIG] product_supplierinfo_for_customer_sale: Migration to 11.0

 - New field name from type to supplierinfo_type
 - Remove copyright for __init__.py
 - Re-using variables from tests
 - Manifest bump version and using module url
 - Get commit history from
 https://github.com/OCA/product-attribute/tree/8.0/product_supplierinfo_for_customer_sale
    - Run 2to3 script
    - view fixing

sergio-teruel committed:
[11.0][NEW] sale_order_general_discount: New module for apply a discount for all sale order lines

pedrobaeza committed:
[IMP] sale_order_general_discount: Improve fields_view_get

moylop260 committed:
[REF] product_supplierinfo_for_customer_sale: Use product.name_search instead of manual assign.

 - Removing duplicated test for name_search

moylop260 committed:
[REF] product_supplierinfo_for_customer_sale: Adding Vauxoo author

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update product_supplierinfo_for_customer_sale.pot

oca-transbot committed:
Update translation files

Updated by Actualizar ficheiros PO com o novo POT (msgmerge) hook in Weblate.

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (23 of 23 strings)

Translation: sale-workflow-10.0/sale-workflow-10.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-10-0/sale-workflow-10-0-sale_cancel_reason/pt/

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

rven committed:
[MIG] sale_order_general_discount: Migration to 12.0

Commit:
[MIG] sale_cancel_reason: Migration to 12.0

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update sale_cancel_reason.pot

pedrobaeza committed:
[FIX] product_supplierinfo_for_customer_sale: Remove offending test

There's no price computation on the module, so there shouldn't be any test regarding
this topic, or it can fail in integration tests.

murtuzasaleh committed:
[12.0][MIG] product_supplierinfo_for_customer_sale

sergio-teruel committed:
[11.0][FIX] product_supplierinfo_for_customer_sale: Search supplierinfo by product variant first

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_cancel_reason/

hugho-ad committed:
[FIX] product_supplierinfo_for_customer_sale: TypeError: unorderable
types: int() >= NoneType()

marylla committed:
Added translation using Weblate (German)

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (23 of 23 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_cancel_reason/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 11.1% (1 of 9 strings)

Translation: sale-workflow-11.0/sale-workflow-11.0-product_supplierinfo_for_customer_sale
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-11-0/sale-workflow-11-0-product_supplierinfo_for_customer_sale/de/

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

Tonow-c2c committed:
[MIG][11] rm coding* + python 2to3

[MIG][11] Update Readme + Manifest

[MIG][11] Adapt procurement for V11 + rm api.one

Tonow-c2c committed:
[MIG][11] Run black to fix lint

Tonow-c2c committed:
[MIG][11] Fix tests for V11 stock.picking

sergio-teruel committed:
[IMP] sale_order_general_discount: Apply general discount to order lines created without UI form view

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (23 of 23 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_cancel_reason/zh_CN/

vrenaville committed:
[FIX] can switch from manual on sent state + fix field name changed

OCA-git-bot committed:
product_supplierinfo_for_customer_sale 11.0.1.0.1

MiquelRForgeFlow committed:
[IMP] Split of 'product.supplierinfo' for 'product.customerinfo'

sudhir-erpharbor committed:
[MIG] sale_order_general_discount: Migration to 13.0

[MIG] Migrated sale_order_general_discount to v13.

[IMP] Improved code as per request.

[IMP][13.0] sale_order_general_discount: rebase with OCA/13.0 + apply pre-commit

[UPD] sale_order_general_discount: update view to display percentage correctly

OCA-git-bot committed:
[UPD] README.rst

grindtildeath committed:
Remove unneeded calls to invalidate_cache

Fix view name and improve tests

IMP: Use a procurement group according to date planned

Fix manifest keys

AaronHForgeFlow committed:
[IMP]allow to manually deliver sale order lines from different sale orders

AaronHForgeFlow committed:
[MIG]sale_manual_delivery to v12

[IMP]sale_manual_delivery. Specify route at the time of the delivery

Commit:
[12.0][FIX] Sale Order Cancel Reasons menu configuration

OCA-git-bot committed:
sale_cancel_reason 12.0.1.1.0

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update product_supplierinfo_for_customer_sale.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-12.0/sale-workflow-12.0-product_supplierinfo_for_customer_sale
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-product_supplierinfo_for_customer_sale/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_order_archive
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_order_archive/

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 95.7% (22 of 23 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_cancel_reason/es_CL/

mmequignon committed:
[FIX] Restore res_partner field in the view with respect to last changes.

Tonow-c2c committed:
[FIX] Partner id will be retrive from wizard to procurement rule

Tonow-c2c committed:
[FIX] wizard float_compare product precision

Tonow-c2c committed:
[FIX] to use the right product uom rounding

OCA-git-bot committed:
pre-commit update

skeller1 committed:
[FIX] sale_cancel_reason: fix access for salesteam salesman group

OCA-git-bot committed:
sale_cancel_reason 12.0.1.2.0

danimv5 committed:
Added translation using Weblate (Spanish)

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_order_archive
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_order_archive/es/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 95.7% (22 of 23 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_cancel_reason/sl/

renatonlima committed:
[FIX] pep8 and pylint

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (6 of 6 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-product_supplierinfo_for_customer_sale
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-product_supplierinfo_for_customer_sale/pt/

maitaoriana committed:
[IMP] sale_order_general_discount: black, isort, prettier

astirpe committed:
[14.0][MIG] sale_order_archive

oca-travis committed:
[UPD] Update sale_order_archive.pot

OCA-git-bot committed:
[UPD] README.rst

alan196 committed:
[IMP] sale_cancel_reason: black, isort, prettier

alan196 committed:
[MIG] sale_cancel_reason: Migration to 13.0

danimv5 committed:
Added translation using Weblate (Catalan)

danimv5 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (7 of 7 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_order_archive
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_order_archive/ca/

bosd committed:
Added translation using Weblate (Dutch)

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 85.7% (6 of 7 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_order_archive
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_order_archive/nl/

sergiocorato committed:
[12.0][IMP] add group to create/edit customerinfo

OCA-git-bot committed:
product_supplierinfo_for_customer_sale 12.0.1.1.0

ivantodorovich committed:
[MIG] sale_order_general_discount: Migration to 14.0

ivantodorovich committed:
[IMP] pre-commit run -a

britoederr committed:
Added translation using Weblate (Portuguese (Brazil))

britoederr committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (7 of 7 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_order_archive
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_order_archive/pt_BR/

oca-travis committed:
[UPD] Update sale_manual_delivery.pot

OCA-git-bot committed:
sale_manual_delivery 12.0.2.0.1

oca-travis committed:
[UPD] Update sale_cancel_reason.pot

Commit:
Translated using Weblate (Dutch)

Currently translated at 100.0% (23 of 23 strings)

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_cancel_reason/nl/

danimv5 committed:
Added translation using Weblate (Catalan)

danimv5 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (6 of 6 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-product_supplierinfo_for_customer_sale
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-product_supplierinfo_for_customer_sale/ca/

danimv5 committed:
Added translation using Weblate (Spanish)

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-product_supplierinfo_for_customer_sale
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-product_supplierinfo_for_customer_sale/es/

brendapaniagua committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (23 of 23 strings)

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_cancel_reason/es/

danimv5 committed:
Added translation using Weblate (Spanish)

danimv5 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (48 of 48 strings)

Translation: sale-workflow-12.0/sale-workflow-12.0-sale_manual_delivery
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-12-0/sale-workflow-12-0-sale_manual_delivery/es/

ivantodorovich committed:
[MIG] sale_manual_delivery: Migration to 13.0

ivantodorovich committed:
[IMP] sale_manual_delivery: black, isort, prettier

oca-travis committed:
[UPD] Update sale_manual_delivery.pot

OCA-git-bot committed:
[UPD] README.rst

vrenaville committed:
[FIX] for drop shipping module

santostelmo committed:
[13.0] Fix view_sales_order_undelivered_line_filter

OCA-git-bot committed:
sale_manual_delivery 13.0.1.0.1

OCA-git-bot committed:
sale_manual_delivery 13.0.1.0.2

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_order_archive
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_order_archive/de/

LoisRForgeFlow committed:
[IMP] product_supplierinfo_for_customer_sale: black, isort, prettier

LoisRForgeFlow committed:
[13.0][MIG] product_supplierinfo_for_customer_sale

JordiBForgeFlow committed:
Clarify the proposal of min_qty in sales order lines

oca-travis committed:
[UPD] Update product_supplierinfo_for_customer_sale.pot

OCA-git-bot committed:
[UPD] README.rst

LoisRForgeFlow committed:
[IMP] product_supplierinfo_for_customer_sale: black, isort, prettier

LoisRForgeFlow committed:
[14.0][MIG] product_supplierinfo_for_customer_sale

oca-travis committed:
[UPD] Update product_supplierinfo_for_customer_sale.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
product_supplierinfo_for_customer_sale 14.0.1.0.1

gurneyalex committed:
[FIX] sale_order_general_discount version

version conflict prevents upload to pypi

vincent-hatakeyama committed:
Translated using Weblate (French)

Currently translated at 95.6% (22 of 23 strings)

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_cancel_reason/fr/

CRogos committed:
[MIG] sale_order_archive : Migrate sale_order_archive to v15.

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-product_supplierinfo_for_customer_sale
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-product_supplierinfo_for_customer_sale/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_order_general_discount
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_order_general_discount/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-product_supplierinfo_for_customer_sale
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-product_supplierinfo_for_customer_sale/it/

ChrisOForgeFlow committed:
[IMP] sale_manual_delivery: black, isort, prettier

ChrisOForgeFlow committed:
[14.0][MIG] sale_manual_delivery

ChrisOForgeFlow committed:
[ADD] init hook script for big databases

ChrisOForgeFlow committed:
[ADD] permissions on trasient models

ChrisOForgeFlow committed:
[IMP] added field to don't show button if not exists pending to deliver

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

CarlosRoca13 committed:
[IMP] product_supplierinfo_for_customer_sale: Add menu entry to access all product.customerinfo

[UPD] Update product_supplierinfo_for_customer_sale.pot

product_supplierinfo_for_customer_sale 15.0.1.1.0

Commit:
[MIG] product_supplierinfo_for_customer_sale: Migration to 15.0

Kev-Roche committed:
[IMP][14.0] Module: sale_order_general_discount test contacts

Kev-Roche committed:
[IMP][14.0] Module: sale_order_general_discount propagate discount to contacts

ao-landoo committed:
[IMP] sale_cancel_reason: black, isort, prettier

ao-landoo committed:
[MIG] sale_cancel_reason: Migration to 14.0

Commit:
[UPD] Update sale_cancel_reason.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_cancel_reason 14.0.1.0.1

ao-landoo committed:
[IMP] sale_cancel_reason: Do not copy cancel reason on order duplicate

ChrisOForgeFlow committed:
[FIX] user validation error, update definition act_window

OCA-git-bot committed:
sale_cancel_reason 14.0.1.1.0

sergio-teruel committed:
[FIX] product_supplierinfo_for_customer_sale: Use the same ACL's to sale module for supplierinfo model
TT38757

Commit:
[UPD] Update product_supplierinfo_for_customer_sale.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update sale_order_archive.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_order_archive 15.0.1.0.1

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_order_archive
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_order_archive/

stefan-tecnativa committed:
[MIG] sale_order_general_discount: Migration to 15.0

Commit:
[UPD] Update sale_order_general_discount.pot

OCA-git-bot committed:
[UPD] README.rst

ferran-S73 committed:
[IMP] sale_order_general_discount: black, isort, prettier

ferran-S73 committed:
[MIG] sale_order_general_discount: migration to 16.0

Reyes4711-S73 committed:
[IMP] product_supplierinfo_for_customer_sale: pre-commit stuff

Reyes4711-S73 committed:
[MIG] product_supplierinfo_for_customer_sale: Migration to 16.0

Commit:
[IMP] sale_order_archive: black, isort, prettier

Commit:
[MIG] sale_order_archive: Migration to 16.0

florian-dacosta committed:
[FIX] Stop deleting lot on sale order cancelation
A lot may have happen, and it is possible to have done some MO, received some PO using this lot, maybe there is even stock on hand with this lot and should not delete it or it is lost everywhere.

Commit:
[IMP] sale_manual_delivery: black, isort, prettier

Commit:
[MIG] sale_manual_delivery: Migration to 16.0

sonhd91 commented on sale_manual_delivery/README.rst:
```diff
@@ -0,0 +1,94 @@
+Known issues / Roadmap
+======================
+
+* Add Screenshots
```
Would be nice if we finished this task hehe
Overall, It's LGTM

mymage committed:
Translated using Weblate (Italian)

Currently translated at 52.1% (12 of 23 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_stock_picking_blocking
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_stock_picking_blocking/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 16.6% (7 of 42 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 19.6% (11 of 56 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_automatic_workflow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_automatic_workflow/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 63.1% (24 of 38 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_type
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_type/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 34.7% (8 of 23 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_cancel_reason/it/

OCA-git-bot committed:
Merge PR #2374 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_order_archive.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_archive
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_archive/

OCA-git-bot committed:
Merge PR #2283 into 16.0

Signed-off-by bguillot

Commit:
[UPD] Update sale_order_lot_generator.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_lot_generator
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_lot_generator/

tate11 committed:
Added translation using Weblate (Turkish)

tate11 committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (19 of 19 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_tier_validation
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_tier_validation/tr/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 52.1% (12 of 23 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_stock_picking_blocking
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_stock_picking_blocking/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 21.4% (12 of 56 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_automatic_workflow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_automatic_workflow/it/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 95.6% (22 of 23 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_cancel_reason/sl/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 65.7% (25 of 38 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_order_type
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_order_type/sl/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 50.0% (28 of 56 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_automatic_workflow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_automatic_workflow/sl/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 16.6% (7 of 42 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/sl/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 52.1% (12 of 23 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_stock_picking_blocking
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_stock_picking_blocking/sl/

OCA-git-bot committed:
Merge PR #2332 into 16.0

Signed-off-by moylop260

Commit:
[UPD] Update product_supplierinfo_for_customer_sale.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-product_supplierinfo_for_customer_sale
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-product_supplierinfo_for_customer_sale/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 34.7% (8 of 23 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_cancel_reason/it/

Commit:
[IMP] sale_cancel_reason: pre-commit stuff

Commit:
[MIG] sale_cancel_reason: Migration to 16.0

Commit:
[MIG] sale_cancel_reason: fix test and pre-commit

Gelo-fl committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_automatic_workflow_payment_mode
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_automatic_workflow_payment_mode/es/

Commit:
[UPD]sale_cancel_reason change setUp to setUpClass

rousseldenis committed:
Translated using Weblate (French)

Currently translated at 78.5% (33 of 42 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/fr/

OCA-git-bot committed:
Merge PR #2354 into 16.0

Signed-off-by rousseldenis

Commit:
[UPD] Update sale_company_currency.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

glitchov committed:
Translated using Weblate (French)

Currently translated at 78.5% (33 of 42 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/fr/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_company_currency
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_company_currency/

OCA-git-bot committed:
Merge PR #2318 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_order_general_discount.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
Merge PR #2483 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_cancel_reason.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_cancel_reason/

Commit:
[16.0][IMP]sale_cancel_reason: set reason to false when status is != cancel

OCA-git-bot committed:
Merge PR #2490 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
sale_cancel_reason 16.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md
